### PR TITLE
Add alias to Sections

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?
- Adds `alias` field to `Sections`
- https://trello.com/c/fvYKd06w/571-aliases-for-sections-9

### How to review 
- Run tests
- Use eQ Author GraphQL API explorer to test mutations and queries eg.

```
mutation CreateSection($input: CreateSectionInput!) {
  createSection(input: $input) {
    alias
  }
}

#Query Variables
{
  "input": {
    "title": "fooBarTitle",
    "alias": "fooBarAlias",
    "questionnaireId": "1"
  }
}
```

```
mutation UpdateSection($input: UpdateSectionInput!) {
  updateSection(input: $input) {
    alias
  }
}

#Query Variables
{
  "input": {
    "id": "1",
    "alias": "fooBarAlias"
  }
}
```

```
query GetSection($id: ID!) {
  section(id: $id) {
    title
    displayName
    alias
  }
}

#Query Variables
{
  "id": "1",
  "alias": "fooBarAlias"
}
```

### Dependencies 

- [x] https://github.com/ONSdigital/eq-author-graphql-schema/pull/65

### CODE REVIEW

**Comment by adammahmood** [comment]
[Backend changes for aliases](https://trello.com/c/5pdOqrqA/612-backend-changes-for-aliases)

File: repositories/SectionRepository.test.js [hunk 1]
```diff
@@ -62,7 +63,8 @@ describe("SectionRepository", () => {
 
     await SectionRepository.update({
       id: result.id,
-      title: "updated title"
+      title: "updated title",
+      alias: "updated alias"
     });
     const updated = await SectionRepository.getById(result.id);
```
Comment (tgandrews) [inline repositories/SectionRepository.test.js]: We should assert that alias is being updated.

File: repositories/SectionRepository.test.js [hunk 2]
```diff
@@ -56,17 +57,20 @@ describe("SectionRepository", () => {
...
+      id: section.id,
+      title: "updated title",
+      alias: "updated alias"
+    };
+
+    await SectionRepository.update(update);
+    const result = await SectionRepository.getById(section.id);
 
-    expect(updated.title).not.toEqual(result.title);
+    expect(result).toEqual(expect.objectContaining(update));
```
Comment (tgandrews) [inline repositories/SectionRepository.test.js]: Can we use `toMatchObject`?

Comment (adammahmood) [reply repositories/SectionRepository.test.js]: Yes. This works despite the misleading name.

File: migrations/20180918144616_add_alias_to_sections.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+exports.up = async function(knex) {
+  await knex.schema.table("Sections", table => table.string("alias"));
+  await knex.raw(`DROP VIEW "SectionsView"`);
+  await knex.raw(`
```
Comment (SamGodwin2) [inline migrations/20180918144616_add_alias_to_sections.js]: Can we return the Promise here, not 100% sure it matters but I think things can sometimes go wrong when we don't return a promise from a knex migration.

**Pull request merged**